docs/source/models/causes/neonatal/preterm_birth.rst

```diff
@@ -165,14 +165,34 @@ The way these CSMRs are used is the same for all subcauses, and therefore is inc
     \text{CSMR}^{k}_{\text{BW},\text{GA}}
     &=
     \begin{cases}
-    \text{CSMR}\cdot f_k \cdot \text{RR}_{\text{BW},\text{GA}} \cdot Z, & \text{if GA} < 37; \\
+    \text{CSMR} / p_\text{preterm} \cdot f_k \cdot \text{RR}_{\text{BW},\text{GA}} \cdot Z, & \text{if GA} < 37; \\
     0, & \text{if GA} \geq 37;
     \end{cases}
     \end{align*}
 
 where :math:`k` is the subcause of interest (preterm birth with or without RDS),
 :math:`\text{CSMR}` is the cause-specific mortality rate for preterm birth complications,
-:math:`f_k` is the fraction of preterm deaths due to subsubcause :math:`k` (with or without RDS), :math:`\text{RR}_{\text{BW},\text{GA}}` is the relative risk of all-cause mortality for a birth weight of :math:`\text{BW}` and gestational age of :math:`\text{GA}`, and :math:`Z` is a normalizing constant selected so that :math:`\int_{\text{GA<37}} \int_{\text{BW}} \text{RR}_{\text{BW},\text{GA}} \cdot Z = 1`.
+:math:`f_k` is the fraction of preterm deaths due to subsubcause :math:`k` (with or without RDS), :math:`\text{RR}_{\text{BW},\text{GA}}` is the relative risk of all-cause mortality for a birth weight of :math:`\text{BW}` and gestational age of :math:`\text{GA}`, and :math:`Z` is a normalizing constant selected so that :math:`Z = 1 / (\overline{\text{RR}_{\text{BW},\text{GA}}} | \text{GA<37})`.
```

Comment:
I'd suggest restoring a version where you haven't yet solved for Z and making this version a second sentence, as the unsolved version makes it clearer why we call it a "normalizing constant."

Also I'm not familiar with this conditional-overline syntax, maybe use $\mathbb{E}[X|Y]$?

Reply:
As in:

```suggestion
:math:`f_k` is the fraction of preterm deaths due to subsubcause :math:`k` (with or without RDS), :math:`\text{RR}_{\text{BW},\text{GA}}` is the relative risk of all-cause mortality for a birth weight of :math:`\text{BW}` and gestational age of :math:`\text{GA}`, and :math:`Z` is a normalizing constant selected so that :math:`(\overline{\text{RR}_{\text{BW},\text{GA}}} \cdot Z | \text{GA<37}) = 1`.
Solving for :math:`Z` gives :math:`Z = 1 / (\overline{\text{RR}_{\text{BW},\text{GA}}} | \text{GA<37})`.
```

Reply:
Agreed that $E[X\mid Y]$ would be more standard notation, rather than $(\overline{X}\mid Y)$.